### ISSUE TITLE
DL3008 false positive caused by squishing apt-get -y and -qq options into -yqq

### ISSUE THREAD

**Issue by mzraly** [hadolint/hadolint#188]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
hadolint should not report DL3008 when Dockerfile RUN directive contains apt-get install command that squishes the -y and -qq options into -yqq (or squishes any other single-letter options that take no arguments).

### Actual behavior
hadolint DOES report DL3008 in this case.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.4.0-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
FROM debian:jessie-slim
RUN apt-get update -qq \
    && apt-get install --no-install-recommends -yqq wget=1.16-1+deb8u4 \
    && rm -rf /var/lib/apt/lists/*
```

**Additional environment details (OS, stack version, etc.)**

**Comment by zemanlx** [comment]
@mzraly FYI
> **-q, --quiet**
Quiet. Produces output suitable for logging, omitting progress indicators. More q's will produce more quiet up to a maximum of two. You can also use -q=# to set the quiet level, overriding the configuration file. **Note that quiet level 2 implies -y**, you should never use -qq without a no-action modifier such as -d, --print-uris or -s as APT may decided to do something you did not expect.

https://linux.die.net/man/8/apt-get
